cdap-docs/_common/common-build.sh

```diff
@@ -75,32 +69,24 @@ else
   PROJECT_PATH="${SCRIPT_PATH}/../../../${2}"
 fi
 
-SDK_JAVADOCS="${PROJECT_PATH}/target/site/${APIDOCS}"
+API_JAVADOCS="${PROJECT_PATH}/target/site/${APIDOCS}"
 
 CHECK_INCLUDES="false"
-TEST_INCLUDES_LOCAL="local"
-TEST_INCLUDES_REMOTE="remote"
-if [ "x${3}" == "x" ]; then
-  TEST_INCLUDES="${TEST_INCLUDES_REMOTE}"
+
+if [[ "${OSTYPE}" == "darwin"* ]]; then
```

Comment:
This should be controlled by a flag, not the operating system where the code is being run. Don't assume that there may never be Darwin-based build hosts (we currently have one).